### PR TITLE
[PBFIG-194] Design Guidelines: Typography

### PR DESCRIPTION
**Pull request by kristiflin**
[PBFIG-194](https://nitro.powerhrg.com/runway/backlog_items/PBFIG-194)
Design Finalized in [PBFIG-161](https://nitro.powerhrg.com/runway/backlog_items/PBFIG-161)

**What does this PR do?** Create a new page in Designer Guidelines

**Screenshots:**
Screenshot 2024-03-28 at 2 08 36 PM

**How to test?** Steps to confirm the desired behavior:
1. Go to  https://pr3298.playbook.beta.hq.powerapp.cloud/guides/design_guidelines/typography_guidelines

#### Checklist:
- [ ] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [ ] **TESTS** I have added test coverage to my code.

### CODE REVIEW

**Comment by thestephenmarshall** [comment]
Correct URL for testing is: https://pr3298.playbook.beta.hq.powerapp.cloud/guides/design_guidelines/typography_guidelines

File: playbook-website/app/views/guides/design_guidelines/typography_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+‣ [Text Modifiers](#Text-Modifiers)
+‣ [Typography Kits](#Typography-Kits)
+‣ [Typography in Tables](#Typography-in-Tables)
+[Typography Accessibility Standards](#Typography-Accessibility-Standards)
+
+Typography serves as the backbone of any design system. It’s not just page titles and copy, typography is everywhere, in navigation labels, buttons, captions, form fields and more. It is essential for providing context, guiding users and delivering the core content of an application.
+
+Playbook’s typography plays a crucial role in shaping the user experience. As a design system primarily used for business software applications, we know that readability is vital for effortless navigation and easy comprehension. To achieve this Playbook uses a professional, unembellished san-serif front and prioritizes intuitive hierarchy in our typography styles. 
+
+When swapping out the default Playbook font, we recommend using a simple sans-serif font such as Proxima Nova or Ariel.
```
Comment (thestephenmarshall) [inline playbook-website/app/views/guides/design_guidelines/typography_guidelines.md:23]:
```suggestion
When swapping out the default Playbook font, we recommend using a simple sans-serif font such as Proxima Nova or Arial.
```

File: playbook-website/app/views/guides/design_guidelines/typography_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+[Typography Usage & Best Practices](#Typography-Usage-amp-Best-Practices)
+‣ [Typography Styles](#Typography-Styles)
+‣ [Creating Visual Hierarchy with Typography](#Creating-Visual-Hierarchy-with-Typography)
+‣ [Text Modifiers](#Text-Modifiers)
+‣ [Typography Kits](#Typography-Kits)
+‣ [Typography in Tables](#Typography-in-Tables)
+[Typography Accessibility Standards](#Typography-Accessibility-Standards)
+
+Typography serves as the backbone of any design system. It’s not just page titles and copy, typography is everywhere, in navigation labels, buttons, captions, form fields and more. It is essential for providing context, guiding users and delivering the core content of an application.
```
Comment (thestephenmarshall) [inline playbook-website/app/views/guides/design_guidelines/typography_guidelines.md:19]:
```suggestion
Typography serves as the backbone of any design system. It’s not just for page titles and copy; typography is everywhere, from navigation labels to buttons, captions, form fields, and more. It is essential for providing context, guiding users, and delivering the core content of an application.
```

File: playbook-website/app/views/guides/design_guidelines/typography_guidelines.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+Playbook offers a range of ready-to-use typography layouts that prioritize displaying clear and concise information.   
+Designed to make the best use of the library’s typography options, layouts offered can be used for date and time formats, contact details, addresses, and label/value groupings. 
+Through strategic use of color, font size, and font weight, Playbook establishes strong informational and visual hierarchies, ensuring effective communication and enhanced readability.
+
+<img src="https://github.com/powerhome/playbook/assets/48187916/550ddd79-932f-4245-82c1-09b02de35345" alt="Tabular Number Spacing">
+
+### Typography in Tables
+
+When working with complex tables, the UI is susceptible to becoming information dense and harder to read. Use the following best practices to create clear, readable tables.
```
Comment (thestephenmarshall) [inline playbook-website/app/views/guides/design_guidelines/typography_guidelines.md:181]:
```suggestion
When working with complex tables, the UI is susceptible to becoming information-dense and harder to read. Use the following best practices to create clear, readable tables.
```

File: playbook-website/app/views/guides/design_guidelines/typography_guidelines.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+
+</div>
+
+<img style="margin:auto; display: block;" src="https://github.com/powerhome/playbook/assets/48187916/4dbcaace-0ce9-43b2-b6f8-9534f278d2b7" width="750" alt="Example table UI">
+
+## Typography Accessibility Standards
+
+#### Color Accessibility Standards
+
+Accessibility in typography is critical to ensuring inclusivity in usability, as text is such atn integral part of any interface.
```
Comment (thestephenmarshall) [inline playbook-website/app/views/guides/design_guidelines/typography_guidelines.md:204]:
```suggestion
Accessibility in typography is critical to ensuring inclusivity in usability, as text is such an integral part of any interface. 
```

**Review by thestephenmarshall — COMMENTED** [review]
Some spelling and grammar nits.

**Review by jasoncypret — DISMISSED** [review]

**Review by nidaqg — APPROVED** [review]

**Comment by jasperfurniss** [comment]
Closed in favor of https://github.com/powerhome/playbook/pull/3346 (verified commits)

**Pull request closed without merging**